xml/Menu/manualdirectdebit.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<menu>
+    <item>
+        <path>civicrm/admin/options/direct_debit_codes</path>
+        <title>Direct Debit Codes</title>
+        <desc></desc>
```

Comment:
please remove the <desc> tag since it is not used. actually, you are already using hook_civicrm_navigationMenu so I don't see a reason to add this xml file, it can be removed.